OPCB/comp/select.sqf

```diff
@@ -65,72 +48,65 @@ _current_tasknumber = format ["TaskNumberFinal_%1",_tasknumber];
 
 _handle = [_radius] spawn CHAB_fnc_findSpot;
```

Comment:
if I remember correctly,  I was using spawn because if I used call it would freeze the server until it finds a good spot. 
it was funny seeing people teleport during mission requests in the early stages tho.

Reply:
We could try again and see if it happens again? :D